src/Microsoft.Azure.WebJobs.Extensions.Kafka/Microsoft.Azure.WebJobs.Extensions.Kafka.csproj

```diff
@@ -7,6 +7,16 @@
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.Kafka</RootNamespace>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Kafka</PackageId>
     <Description>Microsoft Azure WebJobs SDK Kafka Extension</Description>
+    <CommitHash Condition="$(CommitHash) == ''">N/A</CommitHash>
+    <InformationalVersion>$(Version) Commit hash: $(CommitHash)</InformationalVersion>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft</Company>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageLicenseUrl>http://www.microsoft.com/web/webpi/eula/aspnetcomponent_rtw_enu.htm</PackageLicenseUrl>
```

Comment:
is this the right URL? 
if i try browse to that it comes back with a BlobNotFound exception. 

we have a LICENSE file in our github repo, that was put there by the OSS tool that created the repo. 
I believe we should be using that LICENSE file.

Reply:
You right. However it is the same as the event grid repo.  hmm. 
I search other repo. SignalR. It is link to their LICENSE. I'll fix it. 
https://raw.githubusercontent.com/Azure/azure-functions-signalrservice-extension/dev/LICENSE

Reply:
@ryancrawcour I change it to point our project's LICENSE file.